tools/clang/lib/Frontend/FrontendActions.cpp

```diff
@@ -758,7 +758,8 @@ void HLSLRootSignatureAction::ExecuteAction() {
   }
 
   // Compile the expanded root signature.
-  clang::CompileRootSignature(rootSigString, Diags, SLoc, rootSigVer, rootSigHandle.get());
+  clang::CompileRootSignature(rootSigString, Diags, SLoc, rootSigVer, 
+    hlsl::DxilRootSignatureCompilationFlags::GlobalRootSignature, rootSigHandle.get());
```

Comment:
We actually don't want to pass the GlobalRootSignature flag to this one.  This should allow local root signatures to be compiled by dxc.exe directly, like you can with regular root signatures today (not as part of shader compilation).